GiTinder/Models/UserLanguages.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace GiTinder.Models
+{
+    public class UserLanguages
+    {
```

Comment:
@Riceqrisp this should have a single User and a single Language as property

Reply:
Created User property as a string.